README.md

```diff
@@ -10,17 +10,23 @@
    <a href="https://needle.gg">Website ✨</a> &emsp; <a href="https://needle.gg/invite">Invite Needle 🪡</a> &emsp; <a href="https://needle.gg/chat">Get support 💬</a>
 </div>
 
-## Self-hosting (advanced)
+## 👋 Getting started
+
+The easiest way to start using Needle in your server is to use the hosted instance. [Click here to invite Needle to your Discord server](https://needle.gg/invite)!
+
+## 🛠️ Self-hosting (advanced)
+
+The hosted instance of Needle is customizeable and should be enough for most users. However, if you have special requirements and want to modify the functionality of Needle, you will have to fork the repository and self-host your own instance. This requires programming knowledge and is only for advanced users - support for this will be limited.
```

Comment:
Are we also suggesting to people that they will not receive technical support from the needle team if they do this?

Reply:
Just "limited". They may get some support and pointers, but it shouldn't be expected of us.